drltrace/drltrace.c

```diff
@@ -161,7 +161,8 @@ lib_entry(void *wrapcxt, INOUT void **user_data)
             dr_fprintf(outf, "<invalid memory>");
             /* Just keep going */
         });
-        dr_fprintf(outf, ")");
+        dr_fprintf(outf, ") thread id: "TIDFMT" and return to "PFX"",
```

Comment:
"I have some concerns about the return address. First, it seems like an optional feature not needed for all uses, so maybe it should be under a runtime option."
Ok, will put it under optional feature.
"Second, everything else is symbolic: yet here is a raw address. How does the user translate it to something symbolic without also having sthg like a drmodtrack modules log?"

Well, from my practice of drltrace usage for malware analysis, it seems very important to understand actual address from where a function was called (I can later attach a debugger to check specific function, also I can understand that the same functions were called from different places which is also useful). I would prefer to stay it like that. It would be great if we can say that this address is a heap or previously allocated block of memory.